data/data/nutanix/pre-bootstrap/main.tf

```diff
@@ -0,0 +1,74 @@
+locals {
+  description = "Created By OpenShift Installer"
+}
+
+provider "nutanix" {
+  wait_timeout = 60
+  username     = var.nutanix_username
+  password     = var.nutanix_password
+  endpoint     = var.nutanix_prism_central_address
+  port         = var.nutanix_prism_central_port
+}
+
+resource "nutanix_category_key" "ocp_category_key" {
+  name        = "openshift-${var.cluster_id}"
```

Comment:
Is this similar to tags in AWS? If so, then we should stick with the common pattern of using "kubernetes.io/${var.cluster_id}/owned" as the key.

Reply:
That would not be a valid key format for Nutanix Prism Categories :)

Reply:
How about `kubernetes-io-cluster-${var.cluster_id}` for a key, like is used for gcp?

Reply:
That works.